<launch>

    <group ns="localiser">
        <node pkg="localiser" type="localiser" name="localiser" output="screen" required="true">
            <param name="output_bag" value="$(arg bag_output)" />
            <param name="input_bag" value="$(arg bag_input)" />
            <param name="limit_playback_speed" value="true" />
            <param name="percentage_start" value="10" />
            <param name="percentage_end" value="100" />
            <param name="run_pipeline" value="true" />
        </node>
    </group>
</launch>
